models/nc_news_models.js

```diff
@@ -19,7 +19,14 @@ const selectArticleByID = (id) => {
     })
 }
 
+const selectArticles = () => {
+    return db.query(`SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY articles.created_at DESC;`).then((result) => {
+        return result.rows
```

Comment:
Decent query, have a look at how you can turn that count to a number rather than a string. Its not a huge overhaul/refactor.